src/test/java/pom/Homework22.java

```diff
@@ -0,0 +1,60 @@
+package pom;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework22 extends BaseTest {
+    @Test
+    public void renamePlaylist() {
+        String playlistName = "Daria46";
+//        LoginPage loginPage = new LoginPage(driver);
+//        HomePage homePage = new HomePage(driver) ;
+
+        loginPage.login();
+        homePage.doubleClickFirstPlayList();
+        homePage.enterPlaylistName(playlistName);
+        Assert.assertEquals(playlistName,getPlaylistName()); ;
```

Comment:
You got this problem, please correct it. 

![image](https://github.com/Test-Pro-Bootcamp/A46-QA-Automation-Framework/assets/104761482/8d725038-1ec9-4a48-9225-a73a93c072f8)

Reply:
[](url)
<img width="1440" alt="Screenshot 2023-08-06 at 4 32 57 PM" src="https://github.com/Test-Pro-Bootcamp/A46-QA-Automation-Framework/assets/130292323/fc8dce6b-a468-481a-a07f-aadf3f459062">